mk/repos.mk

```
@@ -0,0 +1,12 @@
+.PHONY: import-repos
+TMPDIR := $(shell mktemp -d)
+
+.PHONY: repos-download
+repos-download: ## Download external repo urls from Image Builder
+	git clone https://github.com/osbuild/image-builder.git --sparse --depth=1 $(TMPDIR)
+	cd $(TMPDIR); git sparse-checkout set distributions/ # reduces amount downloaded
+	go run ./cmd/external-repos/main.go download $(TMPDIR)/distributions/
```

```suggestion
.PHONY: repos-download
repos-download: $(GO_OUTPUT)/external-repos  ## Download external repo urls from Image Builder
	{\
		export TMPDIR="$(shell mktemp -d)" ; \
		git clone https://github.com/osbuild/image-builder.git --sparse --depth=1 "$${TMPDIR}" \
		&& ( cd "$${TMPDIR}"; git sparse-checkout set distributions/ ) \
		&& $(GO_OUTPUT)/external-repos download "$${TMPDIR}/distributions/" \
	; }
```
The above would avoid a temporary directory is created by every make invocation (even if it is not the `repos-download` rule).

---

oh good catch!